### ISSUE TITLE
Transfer learning VGG16

### ISSUE THREAD

**Issue by CHIMECHEFULAMP** [CHIMECHEFULAMP/Housing#2]
import tensorflow
from tensorflow.keras import layers, models
from tensorflow.keras.applications import VGG16
from tensorflow.keras.optimizers import Adam

# Load the VGG16 model without pre-trained weights and the top layers
base_model = VGG16(weights=None, include_top=False, input_shape=(32, 32, 3))

# Add new layers on top of the base model
x = base_model.output
x = layers.Flatten()(x)  # Flatten the output before passing it to Dense layers
x = layers.Dense(512, activation='relu')(x)
x = layers.Dense(256, activation='relu')(x)
x = layers.Dense(3, activation='softmax')(x)  # Final output layer

# Define the new model
model = models.Model(inputs=base_model.input, outputs=x)

# Print model summary

model.summary()

#compile

optimizer= Adam(learning_rate =0.004)
model.compile(optimizer , loss="categorical_crossentropy" ,metrics = ["accuracy"])

#train model

#model.fit(training_set,val_set,epoch=5,batch_size=1))
